aries-site/src/data/wcag/tip.json

```diff
@@ -0,0 +1,512 @@
+    "notes": ""
+  },
+  {
+    "rule": "2.5.8",
+    "label": "Target Size (Minimum)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.1",
+    "label": "Language of Page",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.2",
+    "label": "Language of Parts",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.3",
+    "label": "Unusual Words",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.4",
+    "label": "Abbreviations",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.5",
+    "label": "Reading Level",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.6",
+    "label": "Pronunciation",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.1",
+    "label": "On Focus",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.2",
+    "label": "On Input",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.3",
+    "label": "Consistent Navigation",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.4",
+    "label": "Consistent Identification",
+    "Column1": "conditional",
+    "notes": "Tips must be given the correct aria role consistently"
+  },
+  {
+    "rule": "3.2.5",
+    "label": "Change on Request",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.6",
+    "label": "Consistent Help",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.1",
+    "label": "Error Identification",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.2",
+    "label": "Labels or Instructions",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.3",
+    "label": "Error Suggestion",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.4",
+    "label": "Error Prevention (Legal, Financial, Data)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.5",
+    "label": "Help",
+    "Column1": "not-applicable",
+    "notes": "Tip is used to pass this accessibility check for other components but this doesn't apply to the tip itself."
+  },
+  {
+    "rule": "3.3.6",
+    "label": "Error Prevention (All)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.7",
+    "label": "Redundant Entry",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.8",
+    "label": "Accessible Authentication (Minimum)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.9",
+    "label": "Accessible Authentication (Enhanced)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "4.1.2",
+    "label": "Name, Role, Value",
+    "Column1": "conditional",
+    "notes": "Tips must be given the correct aria role consistently"
```

Comment:
Agree this is conditional
I think we could make an enhancement issue to improve the out of the box behavior. Users can pass `role='tooltip'` through `dropProps` in Tip to pass this rule, but we could enhance Grommet to add this automatically for them.
Can we file an enhancement issue for this?